src/Common/tests/System/IO/Compression/ZipTestHelper.cs

```diff
@@ -255,6 +255,8 @@ private static void ItemEqual(String[] actualList, List<FileData> expectedList,
                 String aName = Path.GetFileName(aEntry);
 
                 var bData = expectedList.Where(f => String.Equals(f.Name, aName, StringComparison.OrdinalIgnoreCase)).FirstOrDefault();
+                if (bData == null)
```

Comment:
If this is only failing on older rel branches then it may be because we're missing some test cleanup things I added more recently.

I'll have to look at the exact failures to be sure. The problem is that the Zip tests have some hardcoded expected file values (`FileData` class) that are sometimes too rigid.